src/components/ProposalsItemFooter.vue

```diff
@@ -33,12 +33,31 @@ const { getRelativeProposalPeriod, formatPercentNumber } = useIntl();
       v-if="
         proposal.quorum &&
         proposal.scores_total &&
-        !proposal.space.plugins?.quorum
+        proposal.quorumType === 'default'
       "
     >
       -
       {{ formatPercentNumber(Number(proposal.scores_total / proposal.quorum)) }}
       {{ $t('quorumReached') }}
     </template>
+    <template
+      v-else-if="
+        proposal.quorum &&
```

Comment:
Seems very repetitive, can't we move them inside the same template and check the type? I'd actually use a computed to cleanup the template

Reply:
will try that tmrw

Reply:
Done done ✅